ambry-router/src/main/java/com/github/ambry/router/GetManager.java

```diff
@@ -292,18 +292,3 @@ class GetBlobOptionsInternal {
     this.ageAtAccessTracker = ageAtAccessTracker;
   }
 }
-
-class GetBlobResultInternal {
-  GetBlobResult getBlobResult;
-  List<StoreKey> storeKeys;
-
-  /**
-   * Construct a GetBlobResultInternal instance.
-   * @param getBlobResult The {@link GetBlobResult} associated with this instance, if there is one..
-   * @param storeKeys The store keys associated with this instance, if there are any.
-   */
-  public GetBlobResultInternal(GetBlobResult getBlobResult, List<StoreKey> storeKeys) {
-    this.getBlobResult = getBlobResult;
-    this.storeKeys = storeKeys;
-  }
-}
```

Comment:
we can also remove GetBlobOptionsInternal's getChunkIdsOnly and solely rely on the GetBlobOptions' OperationType.

Reply:
Let me check and I'll use a separate pr to update this part.